operator/option/config.go

```diff
@@ -322,6 +326,7 @@ func (c *OperatorConfig) Populate() {
 
 	c.AWSReleaseExcessIPs = viper.GetBool(AWSReleaseExcessIPs)
 	c.UpdateEC2AdapterLimitViaAPI = viper.GetBool(UpdateEC2AdapterLimitViaAPI)
+	c.UpdateEC2AdapterLimitViaAPI = viper.GetBool(UpdateEC2AdapterLimitViaAPIv2)
```

Comment:
This will ignore the value set by `UpdateEC2AdapterLimitViaAPI`, in the end we want to know the value of any of both flags. Something like this:

`c.UpdateEC2AdapterLimitViaAPI = viper.GetBool(UpdateEC2AdapterLimitViaAPI) || viper.GetBool(UpdateEC2AdapterLimitViaAPIv2)`

Reply:
Let me push a new version.